src/UpdateWizard.js

```diff
@@ -62,14 +63,18 @@ class UpdateWizard extends InstallWizard {
       // operationProps record all the bits that need to run to update
       // TODO how to deal with install password
       operationProps: undefined,
+
+      encryptKey: '',
```

Comment:
It would seem that `isEncrypted` could be derived from `encryptKey`, e.g. it is exactly the same as the expression: `not isEmpty(encryptKey)`, and thus `isEncrypted` is basically duplicated data that should be removed. 

If I am misunderstanding this, then it is worth a comment in the code explaining why two variables are needed here instead of one.

Reply:
thanks @eapodaca  for the explanation. I will update with comments.